src/render/Renderer.cpp

```diff
@@ -1531,7 +1531,7 @@ bool CHyprRenderer::commitPendingAndDoExplicitSync(PHLMONITOR pMonitor) {
     if (inFD >= 0)
         pMonitor->output->state->setExplicitInFence(inFD);
     auto explicitOptions = getExplicitSyncSettings();
-    if (explicitOptions.explicitEnabled && explicitOptions.explicitKMSEnabled)
+    if (explicitOptions.explicitEnabled && explicitOptions.explicitKMSEnabled && !pMonitor->tearingState.canTear)
```

Comment:
this should be activelyTearing not canTear.

Reply:
I tried that first and it didn't work

Reply:
canTear is definitely wrong, but I don't thing we even use the out fence, can we drop it?